build/cmake/functions.cmake

```diff
@@ -166,6 +166,9 @@ function(wx_set_target_properties target_name is_base)
     endif()
 
     if(wxUSE_UNICODE)
+        if(WIN32)
```

Comment:
Curious, is this really necessary? Defining just `_UNICODE` is supposed to be enough -- even if defining this one won't do any real harm neither.

Reply:
Yes, the Windows SDK headers require `UNICODE`, the c++ headers require `_UNICODE`.
Normally wxWidgets will define both if one of the two is defined (in `platform.h`), and Visual Studio also adds them by default. But `sdk_exe` doesn't include any wxWidgets headers, and MinGW does not automatically define `_UNICODE`, hence this fix.